src/AssetManager/Controller/ConsoleControllerFactory.php

```diff
@@ -2,18 +2,34 @@
 
 namespace AssetManager\Controller;
 
+use AssetManager\Service\AssetManager;
+use Interop\Container\ContainerInterface;
+use Zend\ServiceManager\AbstractPluginManager;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class ConsoleControllerFactory implements FactoryInterface
 {
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $serviceLocator = $serviceLocator->getServiceLocator();
-        $console        = $serviceLocator->get('Console');
-        $assetManager   = $serviceLocator->get('AssetManager\Service\AssetManager');
-        $appConfig      = $serviceLocator->get('config');
+        $console        = $container->get('console');
+        $assetManager   = $container->get(AssetManager::class);
+        $appConfig      = $container->get('config');
 
         return new ConsoleController($console, $assetManager, $appConfig);
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function createService(ServiceLocatorInterface $serviceLocator)
```

Comment:
I'm not sure these are needed as the invoke signature is in the LTS of ZF2.  Removing these and the "implements FactoryInterface" will keep deprication notices from throwing in versions > 2.6.